applications/ExternalSolversApplication/ExternalSolversApplication.json

```diff
@@ -1,7 +1,7 @@
 {
 	"wheel_name": "KratosExternalSolversApplication",
 	"included_modules": ["ExternalSolversApplication"],
-	"included_binaries": ["KratosExternalSolversApplication.cp${PYTHON}-win_amd64.pyd"],
+	"included_binaries": ["KratosExternalSolversApplication.*", "KratosExternalSolversCore.*", "libKratosExternalSolversCore.*"],
```

Comment:
as this app is deprecated for very long already I would remove it from the wheels